Survey/Survey/Logger.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import os
+
+class Logger {
+    static let shared = Logger()
+    private let log: OSLog
+
+    private init() {
+        self.log = OSLog(subsystem: Bundle.main.bundleIdentifier ?? "com.yourapp.default", category: "General")
```

Comment:
this looks chatgpt-generated :) Think about what this line is doing.

Reply:
It is. I will change it when will get a chance. Was busy with my job.

Reply:
Done

Reply:
Hmm I'm not seeing the difference. "com.yourapp.default"? 😄

Reply:
I have updated pull request...